chapter1/challenges/program/lib/password_validator.rb

```diff
@@ -3,5 +3,14 @@
 # - include at least one of these characters: !, @, $, %, &
 
 def valid?(password)
+  if (password.length < 8)
+    return false
+  else
+    if (password =~ /[!@$%&]+/)
```

Comment:
I first thought this might fail if the password was "!12345678" since matching a regex this way returns the index of the match (nil if no match) and you would get a zero. But zero is truthy in Ruby! I tried 
0 && true 
in irb and it gives true!